sdk-google/src/main/java/co/reachfive/identity/sdk/google/GoogleProvider.kt

```diff
@@ -34,44 +50,40 @@ class GoogleProvider : ProviderCreator {
         providerConfig: ProviderConfig,
         sessionUtils: SessionUtilsClient,
         context: Context,
+        sdkConfig: SdkConfig
     ): Provider {
-        return ConfiguredGoogleProvider(providerConfig, sessionUtils, context)
+        return ConfiguredGoogleProvider(providerConfig, sessionUtils, context, sdkConfig)
     }
 }
 
 internal class ConfiguredGoogleProvider(
     private val providerConfig: ProviderConfig,
     private val sessionUtils: SessionUtilsClient,
     private val context: Context,
+    private val sdkConfig: SdkConfig,
 ) : Provider {
     private lateinit var origin: String
     private lateinit var scope: Collection<String>
 
-    private val googleSignInClient: GoogleSignInClient
-
     override val requestCode: Int = REQUEST_CODE
     override val name: String = GoogleProvider.NAME
 
-    init {
-        val gso = GoogleSignInOptions
-            .Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
-            .requestScopes(
-                Scope(Scopes.OPEN_ID),
-                *this.providerConfig.scope.map { Scope(it) }.toTypedArray()
-            )
-            .requestServerAuthCode(providerConfig.clientId)
-            .requestEmail()
-            .build()
-
-        googleSignInClient = GoogleSignIn.getClient(this.context, gso)
-    }
+    private val credentialManager: CredentialManager = CredentialManager.create(this.context)
+    private val md = MessageDigest.getInstance("SHA-256")
+    private val b64Flags = Base64.URL_SAFE + Base64.NO_WRAP + Base64.NO_PADDING
 
     override fun login(origin: String, scope: Collection<String>, activity: Activity) {
         this.origin = origin
         this.scope = scope
 
-        val signInIntent = googleSignInClient.signInIntent
-        activity.startActivityForResult(signInIntent, REQUEST_CODE)
+        // Going through the redirection activity is useless,
+        // this is merely a trick to preserve the Provider interface for now till it is refactored
```

Comment:
What is says: this explains the changes all around in the redirection activity and other providers.